bccsp/pkcs11/impl.go

```diff
@@ -162,13 +162,14 @@ func (csp *impl) KeyImport(raw interface{}, opts bccsp.KeyImportOpts) (k bccsp.K
 // the Subject Key Identifier ski.
 func (csp *impl) GetKey(ski []byte) (bccsp.Key, error) {
 	pubKey, isPriv, err := csp.getECKey(ski)
-	if err == nil {
-		if isPriv {
-			return &ecdsaPrivateKey{ski, ecdsaPublicKey{ski, pubKey}}, nil
-		}
-		return &ecdsaPublicKey{ski, pubKey}, nil
+	if err != nil {
+		logger.Errorf("Key not found using PKCS11: %v", err)
```

Comment:
It seems to me that this shouldn't be logged as an error but as a WARN, given that we continue with the execution

Reply:
Fixed